src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -111,48 +116,20 @@ internal Conversions Conversions
         /// </summary>
         private HashSet<SyntaxTree> _lazyCompilationUnitCompletedTrees;
 
-        public override string Language
-        {
-            get
-            {
-                return LanguageNames.CSharp;
-            }
-        }
+        public override string Language => LanguageNames.CSharp;
```

Comment:
Please, in the future refrain from making stylistic changes in the code unrelated to the feature you are working on.